lib/internal/source_map/prepare_stack_trace.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const debug = require('internal/util/debuglog').debuglog('source_map');
+const { findSourceMap } = require('internal/source_map/source_map_cache');
+const { overrideStackTrace } = require('internal/errors');
```

Comment:
I pulled `prepare_stack_trace.js` into its own file, because it seemed like it wasn't directly related to the source map caching functionality (and `source_map_cache.js` was starting to get a bit confusing).